tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -312,15 +312,19 @@ static PyObject * THPVariable_cuda(PyObject* self, PyObject* args, PyObject* kwa
 {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
-    "cuda(int64_t? device=-1, bool non_blocking=False)",
-    "cuda(int64_t? device=-1, bool async=False)|deprecated"
+    "cuda(Device? device=None, bool non_blocking=False)",
+    "cuda(Device? device=None, bool async=False)|deprecated"
```

Comment:
Not a comment to this PR, just curious: what does the `?` after `Device` mean in here? It does have a default value, so I assume it's something more than an optional arg

Reply:
It means optional, it's not necessary with the `=None`, I just like it for consistency.